src/services/application.js

```diff
@@ -18,8 +18,17 @@ const update = async (userId, options) => {
     if (status !== 'VERIFIED') {
       throw new ForbiddenError('User has already submitted an application.');
     }
+    const {
+      firstName, lastName, levelOfStudy, gender, major, shirtSize, resume,
+    } = options;
```

Comment:
Can you change this to args? We're only passing the args at this point, especially since we're not doing anything special to the resume to call it an "option"

Reply:
Added changes, hope you're happy now